sample_code/clean_unknown_vertices/ExprFunctions.hpp

```diff
@@ -541,6 +541,93 @@ inline string CombineTwoNumbers(int64_t one, int64_t two) {
     }
     return outputList;
   }
+
+  inline void RecoverOneEdge(
```

Comment:
Do you want to record the failed ones so that you can resume or analyze it?

Reply:
Recorded in the log so that we can take a look later